vividus/src/integrationTest/java/org/vividus/action/IntegrationElementAction.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.action;
+
+import org.vividus.ui.action.search.IElementAction;
+import org.vividus.ui.action.search.LocatorType;
+
+public class IntegrationElementAction implements IElementAction
```

Comment:
why is it needed?

Reply:
At least one IElementAction is required to satisfy @Inject Set<IElementAction>, btw just declaring set collection with IElementAction typing doesn't work